installer/terraform/scripts/injectingToJenkinsfile.sh

```diff
@@ -6,15 +8,15 @@ scm_type=$2
 
 inject_bootstrap_variables()
 {
-  cd ./jazz-core
-  file_name=Jenkinsfile*
+  cd ./jazz-core || exit
+  file_name="Jenkinsfile*"
 
   # Record the files in an array: jenkinsfile_arr.
   jenkinsfile_arr=()
   for d in **/*/ ; do
-      file=`find "$d" -type f -name "$file_name"`
+      file=$(find "$d" -type f -name "$file_name")
       if [[ "$file" =~ Jenkinsfile* ]] ; then
-        jenkinsfile_arr=(${jenkinsfile_arr[@]} "$file")
+        jenkinsfile_arr=("${jenkinsfile_arr[@]}" "$file")
```

Comment:
Got one error, `^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): Executing: ["/bin/sh" "-c" "./scripts/injectingToJenkinsfile.sh benscen3-gitlab-alb-1355637415.us-east-1.elb.amazonaws.com gitlab"]
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): sed: can't read builds/delete-serverless-service-build-pack/Jenkinsfile
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): builds/delete-serverless-service-build-pack/Jenkinsfile_Cleanup_CF_Dist: No such file or directory
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): sed: can't read builds/delete-serverless-service-build-pack/Jenkinsfile
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): builds/delete-serverless-service-build-pack/Jenkinsfile_Cleanup_CF_Dist: No such file or directory
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): sed: can't read builds/delete-serverless-service-build-pack/Jenkinsfile
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): builds/delete-serverless-service-build-pack/Jenkinsfile_Cleanup_CF_Dist: No such file or directory
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): sed: can't read builds/delete-serverless-service-build-pack/Jenkinsfile
^[[0m^[[0mnull_resource.postJenkinsConfiguration (local-exec): builds/delete-serverless-service-build-pack/Jenkinsfile_Cleanup_CF_Dist: No such file or directory
^[[0m^[[1mnull_resource.postJenkinsConfiguration: Creation complete after 32s (ID: 536112423785532212)^[[0m^[[0m
`
Seems we need to revert it to, jenkinsfile_arr=(${jenkinsfile_arr[@]} "$file")

Please verify

Reply:
I can't really test or diagnose this locally or on my end, I will remove the quotes, let me know if it fixes this or if there's a different error.

Reply:
ok. I had tested in scenario3 and got this issue.  All the jenkins job failed. Let me re test.

Reply:
Yep you're right, I saw `depends_on = ["null_resource.configureJenkins"]` and misremembered that as being a scenario1-only step.

I'm starting to wonder if it might not be better to create a separate set of Terraform scripts for Scenario1 so we don't need all these conditionals and branching logic just to support a use case most OSS users won't use.

Might be cleaner, not sure.